api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+# Working with groups in Microsoft Graph
+
+Groups are collections of [users](*) and [devices](*) (and more?) who share access to resources in Office365 or in your app. All operations in Microsoft Graph on groups require administrator consent. Using Microsoft Graph, you can perform the following operations and more:
+- create groups
+- check if a user is a member of a group
+- list all groups a user is a member of
+- list the members of a group
+- add and remove members from a group
+- add and remove owners of a group.
+
+**NOTE**
+Groups are only supported for enterprise accounts, and not supported for Microsoft personal accounts.
+
+## Office 365 groups
+Office 365 groups are for people who collaborate on a project or a team. Members share access to Office365 resources including:
+- calendar
+- drive
+- notes
+- plans (beta)
+
+Using the API, your app can access these resources to further drive engaging collaboration scenarios. There are 2 types of Office 365 groups:
+
+| Type | Use case | Shared resources |
+|------------|-------|--------|
+| Outlook group | Mail-driven collaboration | Mail conversations and threads, tasks (beta) |

```

Comment:
Are we sure outlook tasks are available in groups?  I think this is only available on /users or /me.